Pod/Classes/Highlightr.swift

```diff
@@ -116,21 +116,22 @@ open class Highlightr
             return nil
         }
         
-        let returnString : NSAttributedString
+        var returnString : NSAttributedString?
         if(fastRender)
         {
             returnString = processHTMLString(string)!
         }else
         {
-             string = "<style>"+theme.lightTheme+"</style><pre><code class=\"hljs\">"+string+"</code></pre>"
+            string = "<style>"+theme.lightTheme+"</style><pre><code class=\"hljs\">"+string+"</code></pre>"
             let opt: [NSAttributedString.DocumentReadingOptionKey : Any] = [
              .documentType: NSAttributedString.DocumentType.html,
-             .characterEncoding: String.Encoding.utf8
+             .characterEncoding: String.Encoding.utf8.rawValue
              ]
             
-             let data = string.data(using: String.Encoding.utf8)!
-             returnString = try! NSMutableAttributedString(data:data, options: opt, documentAttributes:nil)
-
+            let data = string.data(using: String.Encoding.utf8)!
+            safeMainSync {
```

Comment:
Please follow [Allman style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) for indentation.

Reply:
@raspu , If we use `WebKit` because we use the `DocumentType.html` to render, we should call it in main thread,otherwise it leads to accidental crash,  see [Apple Document](https://developer.apple.com/documentation/foundation/nsattributedstring/1524613-init)

Reply:
Noted, thank you!